src/java/org/jivesoftware/openfire/plugin/rest/utils/LoggingUtils.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2022 Ignite Realtime Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jivesoftware.openfire.plugin.rest.utils;
+
+import org.jivesoftware.openfire.plugin.rest.RESTServicePlugin;
+import org.jivesoftware.util.JiveGlobals;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+
+public class LoggingUtils {
+    private static final Logger AUDIT_LOG = LoggerFactory.getLogger("RestAPI-Plugin-Audit");
+
+    public enum AuditEvent {
```

Comment:
I didn't cross-check, but this feels like we're adding an enum value for pretty much every endpoint? I'm not immediately seeing a better way, but that doesn't sit well with me.

Reply:
I'm not in love with it either, but it's that search for something better that's got me stumped. We could make it more freeform, but then it becomes a description in the progress towards structured logging, and there's no "event id" equivalent.